lib/paper_trail/has_paper_trail.rb

```diff
@@ -638,6 +636,11 @@ def save_version?
         (if_condition.blank? || if_condition.call(self)) && !unless_condition.try(:call, self)
       end
 
+      def add_transaction_id(data)
+        return unless self.class.paper_trail_version_class.column_names.include?("transaction_id")
+        data[:transaction_id] = PaperTrail.transaction_id
+      end
```

Comment:
Nice method extraction. How about `add_transaction_id_to` as a method name? (note the "to"). Also, if it's easy to do this method extraction in a separate commit, that would be great. If not, don't worry about it.